config/application.properties

```diff
@@ -22,6 +22,8 @@ http.port=8080
 http.cors.enabled=false
 http.cors.allowedOrigins=*
 http.cors.allowedMethods=GET,POST,PUT,DELETE,OPTIONS,PATCH
+#This property is required to set the predefined headers
+quarkus.http.cors.headers=x-requested-with,x-forwarded-proto,x-forwarded-host,access-control-allow-origin,access-control-allow-methods,origin,content-type,content-length,accept
```

Comment:
I would move this at the bottom in the Quarkus related configuration section.

Reply:
Done @ppatierno